docker-images/prometheus-cloud/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -ex
+
+export BASE_IMAGE="gke.gcr.io/prometheus-engine/prometheus:v2.35.0-gmp.2-gke.0"
+export APPLICATION="prometheus-cloud"
+/usr/bin/env bash ../prometheus/build.sh
```

Comment:
Following the convention:

```suggestion
export IMAGE="${IMAGE:-sourcegraph/prometheus-cloud}"
../prometheus/build.sh
```